SBOM/spdx.md

```diff
@@ -0,0 +1,240 @@
+                },
+                {
+                    "annotationDate": "2023-06-19T10:49:04.840743Z",
+                    "annotationType": "OTHER",
+                    "annotator": "Tool: alma-sbom 0.0.1",
+                    "comment": "almalinux:albs:build:source:gitURL=https://git.almalinux.org/rpms/bash.git"
+                },
+                {
+                    "annotationDate": "2023-06-19T10:49:04.840770Z",
+                    "annotationType": "OTHER",
+                    "annotator": "Tool: alma-sbom 0.0.1",
+                    "comment": "almalinux:albs:build:source:type=git"
+                },
+                {
+                    "annotationDate": "2023-06-19T10:49:04.840798Z",
+                    "annotationType": "OTHER",
+                    "annotator": "Tool: alma-sbom 0.0.1",
+                    "comment": "almalinux:albs:build:source:gitCommit=1a3ebc6398c31698863ba9b31a9f5effdb425c4f"
+                },
+                {
+                    "annotationDate": "2023-06-19T10:49:04.840826Z",
+                    "annotationType": "OTHER",
+                    "annotator": "Tool: alma-sbom 0.0.1",
+                    "comment": "almalinux:albs:build:source:gitRef=imports/c9/bash-5.1.8-6.el9_1"
+                },
+                {
+                    "annotationDate": "2023-06-19T10:49:04.840865Z",
+                    "annotationType": "OTHER",
+                    "annotator": "Tool: alma-sbom 0.0.1",
+                    "comment": "almalinux:albs:build:source:gitCommitCasHash=70c5a02df7074eada535aa35643f019574f7005ff345fd6f482eca797ae7c1d8"
+                }
+            ],
+            "builtDate": "2023-01-24T09:44:30.238513+00:00Z",
+            "checksums": [
+                {
+                    "algorithm": "SHA256",
+                    "checksumValue": "1bfcf5bf3187ff21db8109d48a0198b92341a9124f63db82cf501d1968ba1374"
+                }
+            ],
+            "downloadLocation": "pkg:rpm/almalinux/bash@5.1.8-6.el9_1?arch=x86_64&upstream=bash-5.1.8-6.el9_1.src.rpm",
+            "externalRefs": [
+                {
+                    "referenceCategory": "SECURITY",
+                    "referenceLocator": "cpe:2.3:a:almalinux:bash:5.1.8-6.el9_1:*:*:*:*:*:*:*",
+                    "referenceType": "cpe23Type"
+                },
+                {
+                    "referenceCategory": "PACKAGE_MANAGER",
+                    "referenceLocator": "pkg:rpm/almalinux/bash@5.1.8-6.el9_1?arch=x86_64&upstream=bash-5.1.8-6.el9_1.src.rpm",
+                    "referenceType": "purl"
+                }
+            ],
+            "filesAnalyzed": false,
+            "name": "bash",
+            "supplier": "Organization: AlmaLinux OS Foundation (cloud-infra@almalinux.org)",
+            "versionInfo": "5.1.8-6.el9_1"
+        }
+    ]
+}
+```
+
+#### SPDX SBOM record of a build
+
+**TBD**
```

Comment:
@m10k can you also provide an example of a SBOM record for a build?

Reply:
I just pushed a commit that includes a build SBOM. I also updated the other SBOM in the RFE, since the format has changed quite a bit with the recent changes to alma-sbom.